src/reducers/realTime.js

```diff
@@ -1,18 +1,30 @@
 import { dict } from './../actions'
 const {ipcRenderer} = window.require('electron')
 
-const {SET_BALANCE, SET_TASKLIST, SET_CONNECTED_PEERS, SET_FOOTER_INFO} = dict
+const {SET_BALANCE, SET_TASKLIST, SET_CONNECTED_PEERS, SET_GOLEM_STATUS, SET_FOOTER_INFO} = dict
 
 const initialState = {
-    balance: [Number(0), Number(0)],
+    balance: [0, 0],
     taskList: [],
-    connectedPeers: Number(0), //Number added for preserve 0 case, otherwise js will behave it like boolean
+    connectedPeers: 0,
+    golemStatus: {
+        status: 'Not Ready',
+        message: 'Not connected'
+    },
     footerInfo: null
 }
 
 let badgeActive = false
 let badgeTemp = 0
 
+
+function nodesString(num) {
+    if (num < 1) return 'No Nodes Connected';
+    let postfix = num != 1 ? 's' : '';
```

Comment:
When possible i try to:
- use `const` over `let`
- use `!==` over `!=`

Both could apply here, but are very minor points.